ZAPD/ZTexture.cpp

```diff
@@ -831,8 +831,19 @@ TextureType ZTexture::GetTextureTypeFromString(std::string str)
 
 	if (str == "rgba32")
 		texType = TextureType::RGBA32bpp;
+	else if (str == "rgba16")
+		texType = TextureType::RGBA16bpp;
 	else if (str == "rgb5a1")
+	{
 		texType = TextureType::RGBA16bpp;
+#ifndef DEPRECATION_OFF
```

Comment:
Minor thing but I would prefer if we had ON defines instead of OFF. Double negatives can sometimes get confusing.

Reply:
Done.